src/Queue/AMQPCreateInfo.php

```diff
@@ -166,6 +179,7 @@ public function toArray(): array
             'prefetch'        => $this->prefetch,
             'queue'           => $this->queue,
             'exchange'        => $this->exchange,
+            'exchangeDurable' => $this->exchangeDurable,
```

Comment:
I think it should be `exchangeDurable` if this arg is meant to be passed to the RR: https://github.com/roadrunner-server/amqp/blob/master/amqpjobs/config.go#L26

Reply:
No, camelCase is used here throughout all attributes. (despite yaml using snake_case).

Reply:
I don't mean yaml, this parameter is passed to the RR, and RR expects it to be as I sent it via the link like all other args in the `toArray` function.

Reply:
You're right, my bad. 😩